Sources/SwiftSyntaxBuilderGeneration/Templates/ExpressibleAsProtocolsFile.swift

```diff
@@ -0,0 +1,109 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import SwiftSyntax
+import SwiftSyntaxBuilder
+
+let expressibleAsProtocolsFile = SourceFile {
+  ImportDecl(
+    leadingTrivia: .docLineComment(copyrightHeader),
+    path: "SwiftSyntax"
+  )
+
+  // See BuildableBaseProtocols why `SyntaxCollection` is excluded here.
+  let expressibleAsTypes =
+    SYNTAX_BASE_KINDS
+      .filter { $0 != "SyntaxCollection" }
+      .sorted() // Make sure that the order is deterministic
```

Comment:
I have added a sort here to make the order deterministic (see also https://github.com/apple/swift-syntax/pull/498#discussion_r920976575).